vars/checkmarxExecuteScan.groovy

```diff
@@ -39,7 +39,7 @@ void call(Map parameters = [:]) {
                 passwordVariable: 'password',
```

Comment:
change to `PIPER_password` then you don't need to pass it as parameter any longer :-)